packages/kit/src/views/Send/pages/SendDataInput/SendDataInputContainer.tsx

```
@@ -756,91 +761,102 @@ function SendDataInputContainer() {
+        <Form.Field
+          name="amount"
+          label={intl.formatMessage({ id: ETranslations.send_amount })}
+          rules={{
+            required: true,
+            validate: handleValidateTokenAmount,
+            onChange: (e: { target: { name: string; value: string } }) => {
+              setIsMaxSend(false);
+              const value = e.target?.value;
+              const valueBN = new BigNumber(value ?? 0);
+              if (valueBN.isNaN()) {
+                const formattedValue = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)